src/records/mod.rs

```diff
@@ -21,45 +21,44 @@ mod b_record;
 mod c_record;
 mod d_record;
 mod e_record;
+mod extension;
```

Comment:
Import ordering is a little wonky, I'll hunt around for some better options to tune this (though this might just be a case of disabling it).